packages/wallets/src/hooks/useWalletAccountSwitcher.ts

```diff
@@ -0,0 +1,23 @@
+import { useCallback } from 'react';
+import { useAuthorize, useWalletAccountsList } from '@deriv/api-v2';
+
+const useWalletAccountSwitcher = () => {
+    const { switchAccount: _switchAccount } = useAuthorize();
+    const { data: walletAccounts } = useWalletAccountsList();
+
+    const switchAccount = useCallback(
+        (loginid: string) => {
```

Comment:
so whats the purpose of having separate wallet switcher?

Reply:
why do we need to override default switchAccount behavriour?
if its only to handle that one single case to set initial account, the it would be better to just have function "get initial account" or "get default account" and provide that default account.

Reply:
Thx, I've removed this file